common/app/services/S3.scala

```diff
@@ -34,6 +34,7 @@ trait S3 extends Logging {
 
       val request = new GetObjectRequest(bucket, key)
       val result = client.getObject(request)
+      log.info(s"S3 got ${result.getObjectMetadata.getContentLength} bytes from ${result.getKey}")
 
```

Comment:
Can we possibly log the time for the request too? Also how is the error case logged?